### ISSUE TITLE
Take-Profits are not working on return due to type issues

### ISSUE THREAD

**Issue by peepopoggers** [fabianboesiger/ftx#69]
This happens when trying to initiate an order with TakeProfit order type.

Error Message:
```
Function Exited: unknown variant `take_profit`, expected one of `market`, `limit`, `stop`, `trailingStop`, `takeProfit` at line 1 column 114
```

**Comment by peepopoggers** [comment]
![image](https://user-images.githubusercontent.com/72892531/137132566-72775bea-274f-4593-a928-5d2de8969edd.png)
Also FTX does not provide any information for this response

**Comment by peepopoggers** [comment]
you might have to make your own function for it

**Comment by dovahcrow** [comment]
If opening is not an issue but getting is, how about the fix in #71
